pkg/riff/commands/stream_create_test.go

```diff
@@ -46,6 +46,15 @@ func TestStreamCreateOptions(t *testing.T) {
 			},
 			ShouldValidate: true,
 		},
+		{
+			Name: "valid stream",
```

Comment:
```suggestion
			Name: "with content type",
```

move after the `no provider` test